operations/invoices.md

```diff
@@ -159,8 +162,8 @@ Note: This operation needs [Authentication](../essential-guide/authentication.md
 | `total` | string | required, max length 255 characters | The final amount due on the invoice after all discounts and taxes. |
 | `subtotal` | string | required, max length 255 characters | The total amount of the invoice before taxes and additional charges. |
 | `tip_amount` | string,null | optional, max length 255 characters | The amount of gratuity or tip added to the invoice. |
-| `created_at` | string | required, max length 25 characters | Created at timestamp in ISO 8601 format. |
-| `updated_at` | string | required, max length 25 characters | Updated at timestamp in ISO 8601 format. |
+| `created_at` | string | required, max length 25 characters | Created at timestamp in RFC 3339 format. |
+| `updated_at` | string | required, max length 25 characters | Updated at timestamp in RFC 3339 format. |
```

Comment:
RFC 3339 is a profile (subset) of ISO 8601. Although our timestamps are equally compatible to ISO 8601 and RFC 3339, I think it makes more sense to define them as `RFC 3339` because it is the `date-time` format of json schema.